trestlebot/cli/commands/create.py

```diff
@@ -0,0 +1,119 @@
+"""
+Module for create-cd create-ssp command for CLI
+"""
+
+import click
+
+from trestlebot.cli.options.create import common_create_options
+
+
+
+
+@click.group(name="create")
+@common_create_options
+def create_cmd(ctx: click.Context, profile_name: str) -> None:
+    """
+    Command leveraged for component definition and ssp authoring in trestlebot.
+    """
+
+    pass
+
+
+# @create_cmd.command(name="compdef", help="command for component definition authoring")
+@create_cmd.command("compdef")
+@click.option(
+    "--compdef-name",
+    prompt="Name of component definition is",
```

Comment:
@hbraswelrh Nit pick, but maybe something like "Enter name of component definition" vs asking it as a fill in the blank.

Reply:
Thank you. Agreed.